e2e/framework/utils/mysql_helper.go

```diff
@@ -0,0 +1,114 @@
+package utils
+
+import (
+	"database/sql"
+	_ "github.com/go-sql-driver/mysql"
```

Comment:
this is needed in this case for mysql driver - we need only the side effects, this is the official driver by go and it what they recommend.